optuna/multi_objective/_hypervolume/wfg.py

```diff
@@ -23,6 +24,14 @@ def _compute(self, solution_set: np.ndarray, reference_point: np.ndarray) -> flo
     def _compute_rec(self, solution_set: np.ndarray) -> float:
         assert self._reference_point is not None
         n_points = solution_set.shape[0]
+        _shape = self._reference_point.shape
+
+        if len(_shape) == 1 and _shape[0] == 2:
+            return _compute_2d(solution_set, self._reference_point)
+
+        if len(_shape) == 2:
```

Comment:
Isn't the length of the shape (ndim) of the reference points always 1?

Reply:
I added this logic because I wasn't confident `self._reference_point` is always 1d ndarray.
If we can restrict the shape of input, it fees good for me.

Reply:
Yes, I think we can assume that the input is 1d.